### PR TITLE
Allow RHSSO to have ns_prefix in namespace

### PR DESCRIPTION
**Pull request by aidenkeating**
## Verification Steps
Run the installer with `-e ns_prefix='openshift-'` and ensure everything still works, including monitoring checks.

### CODE REVIEW

File: roles/namespace/tasks/create.yml [hunk 1]
```diff
@@ -8,5 +8,6 @@
   shell: oc create namespace {{ namespace }}
   when: namespace_exists.rc != 0
 
-- name: Give namespace {{ namespace }} display name {{ che_display_name }}
-  shell: oc annotate --overwrite namespace {{ namespace }} openshift.io/display-name="{{ che_display_name }}"
+- name: Give namespace {{ namespace }} display name {{ display_name }}
```
Comment (aidenkeating) [inline roles/namespace/tasks/create.yml]: @matskiv You might need this patch in your PR also. There were some incorrect vars used here.

File: roles/namespace/tasks/create.yml [hunk 2]
```diff
@@ -8,5 +8,6 @@
...
+  shell: oc annotate --overwrite namespace {{ namespace }} openshift.io/display-name="{{ display_name }}"
+  whem: display_name is defined
```
Comment (matskiv) [inline roles/namespace/tasks/create.yml]: typo^ `whem` => `when`

**Comment by philbrookes** [comment]
do you think we should presume the `-`? 
e.g. `-e ns_prefix=openshift` results in `openshift-namespace` and `-e ns_prefix=""` results in `namespace`?

**Comment by matskiv** [comment]
@philbrookes No, I would leave as is - simple and straight forward. Without auto-magical dash :)

**Comment by maleck13** [comment]
**Verification**
Installation completed succesfully

**Pull request merged**